### DISCUSSION TITLE
关于文件下载这个问题，已经解决

### DISCUSSION THREAD

**Discussion by wwwllg** [Tencent/tquic#45] [General]
重新下载你们的最新代码，再次测试已经OK了。出了问题不可怕，Bug解决掉就好了，程序员的修养还需要提高。

**Comment by wwwllg** [comment]
建议再加一个，设置cc的参数

**Comment by wwwllg** [comment]
在延迟较大的环境下，丢包很少低于1%，下载500MB的文件，PQ>(LQ/XQUIC)>TQUIC,(BBR>BBRV3>COPA>CUBIC)

**Reply by golf-gl** [reply]
你好。按照你提供的信息，我们本地测试了一下：
拓扑：server （tc）======  client
server：demo_server（xquic develop branch，cc=bbr）; tquic_server（tquic develop branch，cc=bbr）
client：http_client（lsquic develop branch）
tc emulator规则1：delay = 200ms; loss = 0.5%; Rate = 16Mbps; flow size = 500MB
tc emulator规则2：delay = 200ms; loss = 0.5%; Rate = 24Mbps; flow size = 500MB
次数：各5次

上述两种场景下，xquic和tquic的速度都非常接近，tquic平均速度都略好（<0.5%）。
能够把你具体的测试细节同步一下吗，我们也在自己的环境中更精准地校验。

**Comment by wwwllg** [comment]
你自己租一个实际的，跨国的VPS，测试下就知道了。

**Comment by yflisa** [comment]
@wwwllg 你说的 PQ 是 PicoQuic吗？
